README.md

```diff
@@ -37,6 +37,7 @@ Service APIs implemented:
 - Simple Notification Service (SNS)
 - Web Application Firewall (WAF)
 - AWS Cost and Usage Report API
+- AWS System Manager (SSM)
```

Comment:
I think this is Systems Manager, though this whole list is a bit of a dumpster fire of inconsistency.